tests/tests_geomstats/test_invariant_metric.py

```diff
@@ -776,3 +776,34 @@ def test_dist_pairwise_parallel(self):
         belongs = Matrices(n_samples, n_samples).belongs(result)
         self.assertTrue(is_sym)
         self.assertTrue(belongs)
+
+    def test_integrated_parallel_transport(self):
```

Comment:
Maybe add a test for the code on the geodesic equation too?

Reply:
This is tested by verifying that the end_point of PT coincides with exp

Reply:
I don't think that testing only the shape of the output is relevant, and can't think of an expected value for `geodesic_equation` alone